.github/workflows/linters.yml

```diff
@@ -0,0 +1,48 @@
+name: Linters
```

Comment:
Please use all the files from [this](https://github.com/microverseinc/linters-config/tree/master/html-css) linter config.